bug-1668861-pref-measure-set-to-default-adoption-impact-of-chang-release-81-83.toml

```diff
@@ -0,0 +1,27 @@
+[experiment]
+reference_branch = "modal-default-browser-notification"
```

Comment:
This shouldn't be necessary unless this is wrong in Experimenter for some reason.